### PR TITLE
MinskyStation for Testing and Abuse IV The Revival

### PR DESCRIPTION
**Pull request by fluffe9911**
Go here to see what this is about: https://github.com/yogstation13/yogstation/pull/2281

Also Now With a Map Picture!!
https://imgur.com/a/cClGnua#x1HuaoL (Half Res)
https://www.dropbox.com/s/sh4zofe7o7ggfel/MinskyStation-1.png?dl=0 (Full Res)

To Do:
1. Fix any misc issues that crop up
2. A certain amount of disposal pipes are still borked prob
3. Profit!

Whats new compared to original map:
1. There is now a laser tag arena
2. The transport pipes have been replaced with quantum teleporters
3. Teleport Room, Aux Base, Nanite Chamber and EVA have now been mapped in
4. A ton of other small tweaks and changes

🆑
add: Minsky Station has been authorized for use during populous shifts..again!
/🆑

### CODE REVIEW

**Comment by oklews** [comment]
I love how the map looks so far, from the pictures at least.

**Comment by Mqiib** [comment]
Atmos APCs are a little funky, because when the grid check hit power adamantly refused to return despite us rebooting all the APCs we could find

**Comment by Toxici11i** [comment]
Signal Technician's id cannot open the doors leading in and out of his room ect

**Comment by Sephka** [comment]
I didn't play AI, but heard reports of the AI being unable to track people who were clearly visible on cams. If this hasn't been looked at already, this might want to be reviewed.

**Comment by Toxici11i** [comment]
Telecomms doesnt spawn with a server control machine so signal technicians cannot do their job. The maint access they are supposed to use sometimes doesnt work with their ID so they need an admin to teleport them and spawn the console.

**Comment by Jumps0** [comment]
Played on this earlier, just a few problems in cargo i saw:
-There are dev textures (purple/black) under one of the shutters to the storage area
-ALL of the displays which tell when the supply shuttle arrives don't work
-Below the storage area there is a door that is built in a wall (door opens to wall)
-Both conveyors leading to the supply shuttle when turned out move things away from it (normally one belt leads in, one leads out)
That is all I have noticed in cargo atm, great map, just needs improvement/fixes

**Comment by SomeguyManperson** [comment]
door remotes don't work *scream

File: code/game/objects/structures/crates_lockers/crates/wooden.dm [hunk 1]
```diff
@@ -15,4 +15,5 @@
 	new	/obj/item/reagent_containers/food/drinks/soda_cans/canned_laughter(src)
 	new /obj/item/pneumatic_cannon/pie(src)
 	new /obj/item/reagent_containers/food/snacks/pie/cream(src)
-	new /obj/item/storage/crayons(src)
\ No newline at end of file
+	new /obj/item/storage/crayons(src)
+	new /obj/item/bikehorn/rubber_pigeon(src) //yogs a single rubber pigeon
```
Comment (Altoids1) [inline code/game/objects/structures/crates_lockers/crates/wooden.dm:19]: 🤔

Comment (fluffe9911) [reply code/game/objects/structures/crates_lockers/crates/wooden.dm:19]: The rubber pigeon does not die so easily!

**Pull request merged**